CIP-0029/CIP-0029.md

```diff
@@ -0,0 +1,103 @@
+---
+CIP: 29
+Title: Phase-1 Monetary Scripts Serialization Formats
+Authors: Matthias Benkort <matthias.benkort@iohk.io>
+Comments-URI: https://github.com/cardano-foundation/CIPs/pulls/117
+Status: Draft
+Type: Standards
+Created: 2020-08-17
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This specification describes how to serialize Phase-1 monetary scripts (a.k.a. _"native scripts"_) to various formats (JSON, CBOR) to facilitate inter-operability between applications. 
+
+# Motivation
+
+While the existence of scripts is well-known, there's no agreed upon format when it comes to how those scripts shall be shared across applications. Plus, the on-chain representation of such script isn't really documented anywhere at the moment which makes it challenging for DApps to use them. 
```

Comment:
The on-chain representation is documented in the usual place: the CDDL for each era.

See the table with the CDDL links for each era:
https://github.com/input-output-hk/cardano-ledger-specs#cardano-ledger

Reply:
Yup, my bad. As mentioned in another comment, I completely missed it from the cddl, I think I looked at an old cddl from an old commit hash.